### ISSUE TITLE
createIcons script needs update for 1.21.4

### ISSUE THREAD

**Issue by supercrafter100** [jacobsjo/mcicons#1]
The error I'm getting is the following:
```
Error: Item model for minecraft:stone does not exist
```
This is happening here https://github.com/jacobsjo/mcicons/blob/110b462c96cadca7e5d7752aae267faf6e3be897/src/createItemIcons.js#L58

**Comment by jacobsjo** [comment]
This is happening because of the recent resource-pack changes in 1.21.4. The script always uses assets from the newest snapshot, which is now causing problems. 

The implementation of the new system is currently done in misode/deepslate#41

If you want to run the current script with 1.21.3 assets, you should be able to change https://github.com/jacobsjo/mcicons/blob/110b462c96cadca7e5d7752aae267faf6e3be897/src/createItemIcons.js#L6 to 
```ts
const MCMETA = 'https://raw.githubusercontent.com/misode/mcmeta/refs/tags/1.21.3-'
```

I'll keep this issue open to track the process of updating this script once deepslate is updated.

**Comment by supercrafter100** [comment]
Alright appreciate the response

**Comment by jacobsjo** [comment]
Fixed in b4ea09c91ec479f34cf9a5b5538bb4669bd14b87
